presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -52,6 +52,10 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: ST_LineString(array<Point>) -> LineString
+
+    Returns a LineString from an array of points created with ST_Point.
```

Comment:
Let's drop `created with ST_Point` and clarify that single-element arrays will be rejected.